### ISSUE TITLE
Playback of live content keeps jumping back and forth

### ISSUE THREAD

**Issue by drosalmez** [shaka-project/shaka-player#7391]
**Have you read the [FAQ](https://bit.ly/ShakaFAQ) and checked for duplicate open issues?**
Yes

**If the problem is related to FairPlay, have you read the tutorial?**
N/A, no FairPlay

**What version of Shaka Player are you using?**
Latest: 4.11.7

**Can you reproduce the issue with our latest release version?**
Yes

**Can you reproduce the issue with the latest code from `main`?**
Yes

**Are you using the demo app or your own custom app?**
Basic demo app: https://drosalmez.github.io/shaka_player_demo/

**If custom app, can you reproduce the issue using our demo app?**
Yes

**What browser and OS are you using?**
WPE 2017 browser on Linux

**For embedded devices (smart TVs, etc.), what model and firmware version are you using?**
Proprietary firmware

**What are the manifest and license server URIs?**
See demo app above.

**What configuration are you using?  What is the output of `player.getNonDefaultConfiguration()`?**
See demo app above.

**What did you do?**
Load the above demo app on the WPE browser.

**What did you expect to happen?**
Sample BBC live content plays back smoothly.

https://github.com/user-attachments/assets/5a6091f7-b13a-42a7-a309-b2f4af2128a1

**What actually happened?**
Playback keeps jumping back and forth. See attached recording. The same behaviour can be reproduced with all the live content streams I have tested.

**Are you planning send a PR to fix it?**
?

**Comment by joeyparrish** [comment]
I see the issue in the recording, but I can't reproduce it in a normal desktop browser (ChromeOS 128).  I tried your demo link, and I tried throttling the network to trigger track changes, but nothing I did triggered the bug.

Does this only happen in WPE 2017?

If so, how would I get this version specifically?  My version of Ubuntu seems to have "2.36.0-2" available, so I'm not sure how to compare that to "WPE 2017".
